src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -58,7 +60,8 @@ const createPartyEntity = (
   taxCode: string,
   zipCode: string,
   userRole: UserRole = 'ADMIN',
-  origin: string = 'IPA'
+  origin: string = 'IPA',
+  istat_code: string = 'istat111'

```

Comment:
you want to achieve this? all Parties have the same istat_code with value istat111 if you do this